main.go

```diff
@@ -170,6 +170,11 @@ func main() {
 		log.Fatal("Please specify a config file using --config CONFIG_FILE")
 	}
 
+	jiraPassword := os.Getenv("JIRA_PASSWORD")
```

Comment:
`jiraPassword` still has to be added to the cfg struct :)

Reply:
Sorry @zerok. You shouldn't work on those things when you're distracted. Should be fine now I hope.